sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationproviderbase.py

```diff
@@ -106,8 +108,19 @@ def update_correlation_context_header(
     if is_failover_request:
         correlation_context += ",Failover"
 
+    features = ""
+
     if uses_load_balancing:
-        correlation_context += ",Features=LB"
+        features += "LB,"
```

Comment:
I think this delimiter should be a `+` instead of a comma for values in the `Features=` portion, follows what we do in .NET and for `FFFeatures` too.